Automattic/wp-calypso
# Marketing Connections: Add connection deleted route for Facebook

Facebook and Instagram require a URL to send a user to if they make a
data deletion request in Facebook or Instagram. For us that is simply
deleting the access tokens.

This adds a route that sends the user to their most recent site context
and opens the service that has been 'deleted'.

I was having trouble with the linting on the `import config from 'config'`
line, so in the end I skipped verification.

#### Testing instructions

With this branch go to `/marketing/connection/google_photos` (or another service slug like `mailchimp`) you should be redirected to `/marketing/connections` with the Google Photos service expanded and scrolled into view.
